internal/pkg/otel/samples_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package otel
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/otelcol"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func TestSamples(t *testing.T) {
+	// Set environment variables required by the sample configurations.
+	// These are just sample values to allow the collector to start.
+	os.Setenv("STORAGE_DIR", t.TempDir())
```

Comment:
In `internal/pkg/otel/samples/windows/managed_otlp/logs_metrics_traces.yml` above the usage of `STORAGE_DIR` has been removed, so I am assuming this will be used by something else ?